packages/nginx.rb

```diff
@@ -3,21 +3,13 @@
 class Nginx < Package
   description 'nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server, originally written by Igor Sysoev.'
   homepage 'http://nginx.org/'
-  version '1.13.7'
-  source_url 'https://nginx.org/download/nginx-1.13.7.tar.gz'
-  source_sha256 'beb732bc7da80948c43fd0bf94940a21a21b1c1ddfba0bd99a4b88e026220f5c'
+  version '1.17.4'
+  source_url 'https://nginx.org/download/nginx-1.17.4.tar.gz'
+  source_sha256 '62854b365e66670ef4f1f8cc79124f914551444da974207cd5fe22d85710e555'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/nginx-1.13.7-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/nginx-1.13.7-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/nginx-1.13.7-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/nginx-1.13.7-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'b69ff9cddf02c21fe1e335d17319297a4070cce6be4a33a7ef723dea1042aebe',
-     armv7l: 'b69ff9cddf02c21fe1e335d17319297a4070cce6be4a33a7ef723dea1042aebe',
-       i686: 'a1edfca5490907b118265216c8a92404691321d7492139a3e469fc474bc81c71',
-     x86_64: '01c5891161b00963b17cc2c60871953f9f8fca18d13c5767c8c64b0d11f25cca',
   })
 
   depends_on 'pcre'
```

Comment:
We can remove the `depends_on 'zlibpkg'` line.

Reply:
Rubyize `system "mkdir -p #{CREW_DEST_PREFIX}/bin"` to `FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"`?